cmd/client-s3.go

```diff
@@ -56,6 +57,16 @@ const (
 	googleHostName = "storage.googleapis.com"
 )
 
+// getNumMultiPartThreads gets the number of multipart threads to be used for mc cp/mirror functions
+func getNumMultiPartThreads() int {
+	multipartThreadStr := os.Getenv("DEFAULT_MULTIPART_THREADS")
+	if multipartThreadStr != "" {
+		numThreads, _ := strconv.Atoi(multipartThreadStr)
```

Comment:
I think we can issue a fatal message here when multipartThread env has a bad format.